acra/configuring-maintaining/tls/_index.md

```diff
@@ -2,3 +2,146 @@
 title: TLS
 bookCollapseSection: true
 ---
+
+# TLS
+
+## Common TLS flags
+
+(common for `acra-connector`, `acra-server`, `acra-translator`)
+
+* `--tls_auth=<mode>`
+
+  Set authentication mode that will be used in TLS connection with AcraConnector and database.
+
+  * `0` — do not request client certificate, ignore it if received
+  * `1` — request client certificate, but don't require it
+  * `2` — expect to receive at least one certificate to continue the handshake
+  * `3` — don't require client certificate, but validate it if client actually sent it
+  * `4` — (default) request and validate client certificate
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--tls_key=<filename>`
+
+  Path to private key that will be used in AcraServer's TLS handshake
+  with AcraConnector as server's key and database as client's key.
+  Empty by default.
+
+* `--tls_cert=<filename>`
+
+  Path to TLS certificate, will be sent to other peer during handshake.
+  Empty by default.
+
+* `--tls_ca=<filename>`
+
+  Path to additional CA certificate for AcraConnector and database certificate validation.
```

Comment:
is this CA used for validating database's certificate? And why only AcraConnector? Server and Translator don't have this flag?

Reply:
Rewrote description for `--tls_{key,ca,cert}` flags a bit. WDYT about new version?

I wrote all these docs based on flags description printed by binaries and in some cases the description is different for same flag, so... some mistakes were made.

Reply:
IMO, from my perspective, now it is better